wgpu/src/backend/webgpu.rs

```diff
@@ -1593,12 +1593,7 @@ impl dispatch::AdapterInterface for WebAdapter {
     fn request_device(
         &self,
         desc: &crate::DeviceDescriptor<'_>,
-        trace_dir: Option<&std::path::Path>,
     ) -> Pin<Box<dyn dispatch::RequestDeviceFuture>> {
-        if trace_dir.is_some() {
-            //Error: Tracing isn't supported on the Web target
-        }
```

Comment:
I see this was lacking to begin with, but I think this should log a warning

Reply:
Done.